opal/mca/common/ucx/common_ucx.c

```diff
@@ -6,6 +6,11 @@
  *
  * $HEADER$
  */
+/*
```

Comment:
These types of headers aren't necessary -- this kind of information is in the git logs.